test/topology_raft_disabled/test_raft_upgrade_basic.py

```diff
@@ -25,7 +25,7 @@
 @pytest.mark.replication_factor(1)
 async def test_raft_upgrade_basic(manager: ManagerClient, random_tables: RandomTables):
     """
-    kbr-: the test takes about 7 seconds in dev mode on my laptop.
+    kbr-: the test takes about 12 seconds in dev mode on my laptop.
```

Comment:
Will you patch it each time your laptop changes? :)

Reply:
Removed the comment